lib/Dist/Zilla/Plugin/Alien.pm

```diff
@@ -330,6 +357,18 @@ sub _helper_hash {
 	\%helper;
 }
 
+has env => (
+	isa => 'ArrayRef[Str]',
+	is => 'rw',
+	default => sub { [] },
+);
+
+sub _env_hash {
+	my($self) = @_;
+	my %env = map { /^\s*(.*?)\s*=\s*(.*)\s*$/ ? ($1 => $2) : ($_ => '') } @{ $self->env };
```

Comment:
No I understand.  I will make the change.  A note in the documentation for the unwary here and in AB will help too.